Sources/Filters/Sources/CubeSource/test/testCube.js

```diff
@@ -85,3 +85,13 @@ test.onlyIfWebGL('Test vtkCubeSource Rendering', (t) => {
   });
   renderWindow.render();
 });
+
+test('Test vtkCubeSource : Same output bounds as inputs', (t) => {
+  const bounds = [
+    -10.2368459, 10.3651298, -10.2569841, 10.8746985, -10.5987456, 10.6549872,
+  ];
+  const cube = vtkCubeSource.newInstance();
+  cube.setBounds(bounds);
+  t.deepEqual(cube.getOutputData().getBounds(), bounds);
```

Comment:
is the test failing if pointType is 'Float32Array' ?

Reply:
Not tested yet because I have issue with these data. I have an error of 10e-9. I suppose that it's correct, isn't it ? That's why it's WIP.

Reply:
error of 10e-9 in the case of `Float64Array` or `Float32Array` ?

Reply:
After deep test, here is the status:
- In float64, I have errors of 10e-14
- In float32, I have errors between 10e-6 and 10e-7

Reply:
If you still have errors in float64, it means that this fix will not be sufficient for our reslice cursor widget problem.
We need to inflate the bounds anyway.
So maybe this entire fix is not mandatory (for our problem) and we can leave everything in float32.

WDYT @martinken @jourdain  ? Do we want to default to 64bit points (and optionally normals+tc) or is it useless because it will be converted back to 32bits for WebGL ?

Reply:
IMO all source/widget points should be float64 unless they have a huge number of points.  Normals and tcoords should be float32 as their ranges are much better behaved in normal usage. 

WebGL and WebGPU backends will convert to float32 but it will use shift scale to best render the float64 data. For geophysical data float32 points are almost always a problem unless some sort of local reference frame is used.

Reply:
FWIW in VTK I have had to  convert a number of sources/widgets to float64 to prevent significant positional jitter. It is very noticeable. Moving to float64 with shiftscale makes it render correctly.

Reply:
Gotcha ! Thanks for your input.
@laurennlam we can therefore keep that commit 'Float32Array' to 'Float64Array'

Reply:
#2237